content/zh/docs/setup/kubernetes/additional-setup/config-profiles/index.md

```diff
@@ -46,11 +46,11 @@ keywords: [profiles,install,helm]
 
 有些配置文件会有一个带有认证功能的变体，文件名会加入 `-auth`，这种配置会加入以下几个功能：
 
-| | default | demo | minimal | sds |
-| --- | :---: | :---: | :---: | :---: |
-| Control Plane Security | | X | | |
-| Strict Mutual TLS | | X | | X |
-| SDS | | | | X |
+| | default | minimal | sds |
```

Comment:
I can't read Chinese, but looking at this table, I'm pretty sure the zh version of this doc is not consistent with the en version.

Reply:
Thanks @frankbu for your findings. Let me double check ensure it is consistence. Will have a PR for it.